### PR TITLE
Remove inline JavaScript from header and replace it with Jekyll directive

### PR DESCRIPTION
**Pull request by GitHugop**
Ref: #16 
Uses site.url and removes the "https://" part.

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name"><script type="text/javascript">document.write(gheHostname())</script> Statistics</h1>
+				<h1 class="project-name">{{ site.url | remove:'https://' }} Statistics</h1>
```
Comment (pluehne) [inline docs/_layouts/default.html]: Could you plase add a space after the colon and put the `https://` part in double quotes for consistency with the rest?

**Comment by pluehne** [comment]
I’m not sure whether it would also be good to entirely get rid of the [`gheHostname` and `gheUrl` JavaScript functions](https://github.com/Autodesk/hubble/blob/master/docs/assets/js/charts.js#L318-L324) still used [here](https://github.com/Autodesk/hubble/blob/master/docs/assets/js/charts.js#L375).

@larsxschneider, @GitHugop: What do you guys think?

File: docs/_layouts/default.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name">{{ site.url | remove:'https://' }} Statistics</h1>
+				<h1 class="project-name"> {{ site.url | remove:"https://" }} Statistics</h1>
```
Comment (pluehne) [inline docs/_layouts/default.html]: Sorry for being ambiguous. I meant that a space should be inserted after `remove:` like this: `remove: "https://"` (and not in the beginning. Could you change that?

I know I’m a bit pedantic, but I want to keep the coding style consistent as much as possible 😃.

File: docs/_layouts/default.html (hunk 1 above)
Comment (GitHugop) [reply docs/_layouts/default.html]: Yes, I have updated the file with your requested changes.

**Comment by pluehne** [comment]
Let’s wait for @larsxschneider to review and merge this.

@larsxschneider: Do you think it would make sense to also evaluate the server name via Jekyll for the `charts.js`, as I suggested above?

**Comment by GitHugop** [comment]
You welcome. Regarding the functions, I don't think it's a great idea to mix JavaScript files with Jekyll. Something that can be done is merging both functions into one, like:

`
function getURL() {
return $(location).prop("protocol") + "//" + $(location).prop("hostname");
}
`

File: docs/_layouts/default.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name"><script type="text/javascript">document.write(gheHostname())</script> Statistics</h1>
+				<h1 class="project-name">{{ site.url | remove: "https://" }} Statistics</h1>
```
Comment (larsxschneider) [inline docs/_layouts/default.html]: Thank you 👍 ! Is there a way to handle `http://` URLs, too? Some GitHub Enterprise installations use just HTTP 😢

Comment (pluehne) [reply docs/_layouts/default.html]: Wasn’t aware of this, good catch!

Comment (GitHugop) [reply docs/_layouts/default.html]: Testing it out.

File: docs/_layouts/default.html [hunk 4]
```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name"><script type="text/javascript">document.write(gheHostname())</script> Statistics</h1>
+				<h1 class="project-name">{{ site.url | remove: "http://", "https://" }} Statistics</h1>
```
Comment (larsxschneider) [inline docs/_layouts/default.html]: I get this error with Jekyll 3.3.1 locally:
```
  Liquid Exception: Liquid error (line 38): wrong number of arguments (given 3, expected 2) in /_layouts/default.html
```
Any idea what to do about it?

Comment (GitHugop) [reply docs/_layouts/default.html]: Seems like "remove" only supports 1 argument, there doesn't seem to be a way on the Jekyll docs. We can try using both "remove" and "remove_first".

**Comment by larsxschneider** [comment]
Thank you @GitHugop ! 👍 👍 👍 

@pluehne Yeah, we could use the same approach in `chart.js`.

**Pull request merged**